app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
OK. So do you think there's a way to conform to the existing URL schema in routes.rb (which demands a non-unique "legacy slug") by somehow concatenating the slug with the date before making the friendly_id-based `find()` call? (and doing this only for maps?) I know this is difficult challenge, but I would really like to merge this new code.